File: Array/yanghao/java/Algorithm6_23.java
package jdbc.mvc.interfaces;

import java.util.List;

public class Algorithm6_23 {

	public int findNumbers(int[] nums) {
		int a = 0;
		for (int i = 0; i < nums.length; i++) {
			String c = String.valueOf(nums[i]);
			if (c.length() % 2 == 0) {
				a++;
			}
		}
		return a;
    }
}

File: Array/yanghao/kotlin/Algorithm6_23.kt
package staudy
class Algorithm6_23 {

  fun findNumbers(nums:Array<Int>) : Int{
      var a = 0
      for (i in 1..nums.size){
        val c = nums[i].toString()
        if (c.length %2  == 0){
          a++
        }
      }
    return a
  }
}